cmake-proxies/cmake-modules/Findlibmp3lame.cmake

```diff
@@ -0,0 +1,30 @@
+#[[
+A module to look for libmp3lame
+]]
+
+if( NOT libmp3lame_FOUND )
+
+    find_path( libmp3lame_INCLUDE_DIR lame/lame.h )
+    find_library( libmp3lame_LIBRARIES NAMES mp3lame )
```

Comment:
This file is using 4 space indentation rather than 3 used in most of Audacity's CMake files. I think there's already a mix of indentation in the repository, but it's something to be aware of when we add new files.